internal/engine/buildcontroller_test.go

```diff
@@ -245,6 +246,51 @@ func TestBuildControllerCrashRebuild(t *testing.T) {
 	f.assertAllBuildsConsumed()
 }
 
+func TestCrashRebuildTwoContainersOneImage(t *testing.T) {
```

Comment:
is it worth testing the behavior when it's two containers on a single pod, each with a different image? should work the same, but maybe good for the sake of coverage.

Reply:
i thought that didn't work yet?

Reply:
OH! yeah that building behavior doesn't work, you're right -- was imaging that you could directly set up the pod/build-result state. but that's probably not worth it. leave a todo for that test tho, and i'll fill it in

Reply:
filed as https://app.clubhouse.io/windmill/story/3095/add-tests-for-live-update-two-containers-with-different-images